content/docs/enterprise/external-data/geoip.mdx

```diff
@@ -1,47 +1,100 @@
 ---
-title: GeoIP Ranges
-draft: true
-description: Configure the $EXT_DATA_PROVIDER external data provider to extend your access policies.
+title: GeoIP
+description: Configure the GeoIP external data provider to extend your access policies.
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-The $EXT_DATA_PROVIDER integration provides {{Fill me in with content specific to this data provider}}.
-
-This dataset provides a list of IP address ranges and their country of origin, according to ARIN:
+The GeoIP integration provides provides a list of IP address ranges and their country of origin:
```

Comment:
Would be nice to give a few concrete examples of how this would be useful in policy. Maybe 3 or so examples? 
```suggestion
The GeoIP integration provides provides a list of IP address ranges and their country of origin. For example, this data provider could allow an administrator to restrict, or allow access based on users' known GeoIP fingerprint.
```

Something like that?

Reply:
```suggestion
The GeoIP integration provides provides a list of IP address ranges and their country of origin. This data provider could allow an administrator to restrict, or allow access based on a users' known GeoIP fingerprint.
```